OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
I'll pass along the conversation.

Right so one way or another the Artillery/Parabomb damage will be sort of "redefined" having  shells with different effects on to different sized/tiled buildings. Smaller is better. If proper changes are done accordingly I don't see why they can't eventually become an integrated, fun and positive balance feature. But that will take time.

With the exception of the SAM Site (2x1, heavy armor), damage vs wood looks like a key value to be looked at as all 1x1+ buildings have wood armor. I'm sure there's countless of ways to look at the issue as well but I always prefer to start simple.

All-righto, here goes nothing :+1: